src/chocolatey/infrastructure.app/services/TemplateService.cs

```diff
@@ -188,5 +196,162 @@ public void generate_file_from_template(ChocolateyConfiguration configuration, T
+                    {
+                        list_built_in_template_info(configuration, isBuiltInTemplateOverriden, isBuiltInOrDefaultTemplateDefault);
+                    }
+                }
+                else
+                {
+                    if (configuration.TemplateCommand.Name.ToLowerInvariant() == _builtInTemplateName || configuration.TemplateCommand.Name.ToLowerInvariant() == _builtInTemplateOverrideName)
+                    {
+                        // We know that the template is not overriden since the template directory was checked
+                        list_built_in_template_info(configuration, isBuiltInTemplateOverriden, isBuiltInOrDefaultTemplateDefault);
+                    }
+                    else
+                    {
+                        throw new ApplicationException("Unable to find requested template '{0}'".format_with(configuration.TemplateCommand.Name));
+                    }
+                }
+            }
+        }
+
+        protected void list_custom_template_info(ChocolateyConfiguration configuration, IPackageManager packageManager)
+        {
+
+            var pkg = packageManager.LocalRepository.FindPackage("{0}.template".format_with(configuration.TemplateCommand.Name));
+            var pkgVersion = (pkg == null) ? "0.0.0" : pkg.Version.to_string();
+            var pkgTitle = (pkg == null) ? "{0} (Unmanaged)".format_with(configuration.TemplateCommand.Name) : pkg.Title;
+            var pkgSummary = (pkg == null) ? 
+                string.Empty : (pkg.Summary != null && !string.IsNullOrWhiteSpace(pkg.Summary.to_string()) ? "{0}".format_with(pkg.Summary.escape_curly_braces().to_string()) : string.Empty);
+            var pkgDescription = (pkg == null) ? string.Empty : pkg.Description.escape_curly_braces().Replace("\n    ", "\n").Replace("\n", "\n  ");
+            var pkgFiles = "  {0}".format_with(string.Join("{0}  "
+                .format_with(Environment.NewLine), _fileSystem.get_files(_fileSystem
+                    .combine_paths(ApplicationParameters.TemplatesLocation, configuration.TemplateCommand.Name), "*", SearchOption.AllDirectories)));
+
+            var isOverridingBuiltIn = configuration.TemplateCommand.Name == _builtInTemplateOverrideName;
+            var isDefault = string.IsNullOrWhiteSpace(configuration.DefaultTemplateName) ? isOverridingBuiltIn : (configuration.DefaultTemplateName == configuration.TemplateCommand.Name);
+            
+            if (configuration.RegularOutput)
+            {
+                if (configuration.Verbose)
+                {
+                    // TODO list parameters, do later (save to file?)
+                    this.Log().Info(@"Template name: {0}
+Version: {1}
+Default template: {2}
+{3}Title: {4}
+{5}{6}
+List of files:
+{7}
+List of Parameters: 
+  Will be implemented at a later time

```

Comment:
Rather than include this here, I think we should remove this, and simply add this functionality in a later issue.  I don't think we need to call out here that this will be added at some point in the future.

Reply:
Done